extensions/model-context-protocol-registry/src/registries/builtin/entries.ts

```diff
@@ -410,6 +410,20 @@ export const OFFICIAL_ENTRIES: RegistryEntry[] = [
       },
     },
   },
+  {
+    name: "apify",
+    title: "Apify",
+    description: "A Model Context Protocol server for Apify, enabling AI assistants to interact with Apify's web scraping and automation platform. This server provides tools to manage actors, runs, and datasets through Large Language Models.",
+    icon: "https://svgl.app/library/apify.svg",
```

Comment:
This link doesn't seem to work, ideally let's use some logos available under apify.com/ext - see https://github.com/apify/apify-web/tree/develop/public/ext, e.g. http://apify.com/ext/apify-symbol-512px.svg